### ISSUE TITLE
Install instructions?

### ISSUE THREAD

**Issue by flaugher** [jeetsukumaran/vim-buffergator#36]
It would be helpful if you could include brief instructions on how to install buffergator for those of us who aren't yet Vim experts.  Thanks.

**Comment by jeetsukumaran** [comment]
Hi @flaugher, you are correct that a little section detailing this should be added. I will be traveling for the next couple of weeks, but will (try to remember to) take care of this when I get back!

**Comment by 7Moses** [comment]
Agreed, I am using Vundle, In Vim :PluginInstall and in Terminal $ vim +PluginInstall +qall! And the line,  Plugin 'vim-buffergator/vim-buffergator' in vimrc, ... so far throwing an error on install. Thanks,

**Comment by 7Moses** [comment]
Made a change,

Using https://github.com/skwp/dotfiles

And installed via

yav -u https://github.com/jeetsukumaran/vim-buffergator, 

Best,

Will test later,

**Comment by flaugher** [comment]
I'm trying to install vim-buffergator on a new Mac and I'm having trouble again.  Here's my .vimrc file:

set nocompatible    " Required by Vundle
filetype off      " Required by Vundle
set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()
Plugin 'gmarik/Vundle.vim'
Plugin 'vim-buffergator'

When I run the command "vim +PluginInstall +qall"  I see this:

Processing 'vim-buffergator'Username for 'https://github.com':

Why is this happening?  I also don't understand your last comment.  What's "yav"?

**Comment by mynamewastaken** [comment]
@flaugher on the off chance this is still a problem, you probably need to give it the username that the buffergator repo is under. e.g.:

`Plugin 'jeetsukumaran/vim-buffergator'`

**Comment by robertf57** [comment]
Yes, I discovered last week that that was the problem.  In not prefacing the plugin with the plugin author's name, Vim thought I was trying to install the plugin from my personal Github account and was prompting me for my password.  When I inserted "jeetsukumaran/" into the Plugin argument, the plugin installed without a problem.  Thanks for your follow-up.
